### ISSUE TITLE
Create sim primer

### ISSUE THREAD

**Issue by arouinfar** [phetsims/gravity-force-lab#97]
- [ ] Draft script
- [ ] Record rough version for script verification 
- [ ] Review script verification draft, and adjust script as necessary 
- [ ] Record another draft
- [ ] Review 2nd draft
- [ ] Produce primer and upload 
- [ ] Caption primer

**Comment by arouinfar** [comment]
Should be done in conjunction with https://github.com/phetsims/gravity-force-lab-basics/issues/67 and https://github.com/phetsims/coulombs-law/issues/56.

**Comment by arouinfar** [comment]
The primer has been recorded and uploaded to Vimeo. Publication to the PhET site should wait until this sim redeploys.

**Comment by terracoda** [comment]
@arouinfar, is it safe to assume that since this is marked "fixed-awaiting-deployment" that there is nothing to be done for the primer for Gravity Force Lab Basics? We are getting ready to publish Basics, so I just wanted to be sure.

**Comment by arouinfar** [comment]
@terracoda that's correct! This issue is awaiting the redeploy of GFL. The GFLB primer is also ready to go, but that's being tracked in in the master checklist.

**Comment by zepumph** [comment]
GFL 2.2.0 was just published. I'll let you do your thing here!

**Comment by arouinfar** [comment]
Unfortunately, the primer was recorded before we added the force values radio button group. This primer will need to be re-recorded. I don't currently have any bandwidth for primers, so marking this as on-hold.

**Comment by arouinfar** [comment]
Sim primers are not currently a priority, unassigning myself.
